webapp/utils/aws/cloudwatch.py

```diff
@@ -0,0 +1,88 @@
+import logging
+from typing import TYPE_CHECKING
+
+import boto3
+from attrs import define, field
+
+if TYPE_CHECKING:
+    from mypy_boto3_logs.client import CloudWatchLogsClient as CloudWatchLogsClientType
+
+from webapp.config import Config
+from webapp.exceptions import ECSTaskLogStreamDoesNotExistError
+
+logger = logging.getLogger(__name__)
+
+
+@define
+class CloudWatchLogsClient:
```

Comment:
Short of getting wired up to routes and logic in the actual flask app, this class looks great!  Simple methods, makes sense.